doc/zephyr.doxyfile

```diff
@@ -113,7 +113,7 @@ INPUT                  = \
 INPUT_ENCODING         = UTF-8
 FILE_PATTERNS          = "*.c" "*.h" "*.S"
 RECURSIVE              = YES
-EXCLUDE                = ../include/spi_legacy.h
+EXCLUDE                = ../include/spi_legacy.h ../include/net/http_legacy.h
```

Comment:
If we're starting a trend to ignore *_legacy.h files, better would be to delete these individual files listed here and change the line below to be:

     EXCLUDE_PATTERNS = *_legacy.h

Reply:
I am ok with this. Perhaps this could be a separate patch and not part of this set.

Reply:
Ok. I’ll submit a separate patch for ignoring all _legacy.h files.